firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/CrashlyticsController.java

```diff
@@ -397,7 +396,7 @@ public Void call() throws Exception {
   void writeNonFatalException(@NonNull final Thread thread, @NonNull final Throwable ex) {
     // Capture and close over the current time, so that we get the exact call time,
     // rather than the time at which the task executes.
-    final Date time = new Date();
+    final long time = System.currentTimeMillis();
```

Comment:
Maybe rename this to timeMs to be consistent with the other changes?

Reply:
Done!